src/pages/dashboard/EventSummary.jsx

```diff
@@ -57,53 +84,40 @@ const EventSummary = () => {
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
           />
-          <button className="bg-[#1070FF] md:px-12 md:py-4 py-2.5 px-5 text-white rounded-lg">
+          <button
+            className="bg-[#1070FF] md:px-12 md:py-4 py-2.5 px-5 text-white rounded-lg"
+            onClick={handleSubmit}
+          >
             Done
           </button>
         </div>
         <div className="my-12">
-          <div className="flex justify-between mb-4">
-            <div className="flex text-[#59595B] items-center">
-              <AiOutlineUser className="text-xl" />
-              <p className="font-normal md:text-base text-sm md:ml-3 ml-2">
-                Damijoshua@gmail.com
-              </p>
-            </div>
-            <div className="flex items-center">
-              <AiOutlineLike className="text-2xl text-[#006600]" />
-              <p className="text-xs ml-1.5">Yes</p>
+          {participant.map((invite, index) => (
+            <div className="flex justify-between mb-4" key={index}>
+              <div className="flex text-[#59595B] items-center">
+                <AiOutlineUser className="text-xl" />
+                <p className="font-normal md:text-base text-sm md:ml-3 ml-2">
+                  {invite.email}
+                </p>
+              </div>
+              <div className="flex items-center md:mr-8">
+                <p className="text-xs ml-1.5">{invite.value}</p>
+                {invite.value === "Yes" ? (
+                  <AiOutlineLike className="text-2xl text-[#006600]" />
+                ) : (
+                  <AiOutlineDislike className="text-2xl text-[#CC0000]" />
+                )}
+              </div>
             </div>
-          </div>
-          <div className="flex justify-between mb-4">
-            <div className="flex text-[#59595B] items-center">
-              <AiOutlineUser className="text-xl" />
-              <p className="font-normal md:text-base text-sm md:ml-3 ml-2">
-                SarahM32492@gmail.com
-              </p>
-            </div>
-            <div className="flex items-center">
-              <AiOutlineDislike className="text-2xl text-[#CC0000]" />
-              <p className="text-xs ml-1.5">No</p>
-            </div>
-          </div>
-          <div className="flex justify-between mb-4">
-            <div className="flex text-[#59595B] items-center">
-              <AiOutlineUser className="text-xl" />
-              <p className="font-normal md:text-base text-sm md:ml-3 ml-2">
-                FaithBala2@gmail.com
-              </p>
-            </div>
-            <div className="flex items-center">
-              <AiOutlineDislike className="text-2xl text-[#CC0000]" />
-              <p className="text-xs ml-1.5">No</p>
-            </div>
-          </div>
+          ))}
         </div>
         <div className="my-6 flex justify-between items-center">
           <Link to="/" className="text-xl font-semibold">
```

Comment:
- [ ] Good job on the responsiveness of the page 👍🏽. I can see that the create event button is currently not linked to the `create event `page. Kindly link the button to route to the c`reate event` page 😄